.rubocop.yml

```diff
@@ -111,10 +107,10 @@ RSpec/MessageSpies:
   EnforcedStyle: receive
 
 RSpec/ExampleLength:
-  Max: 16
+  Max: 31
```

Comment:
I don't think it is a good idea to change the project's rubocop settings. We would need to check with the Hyku project owner before making changes to this file.

Reply:
Thank you so much for this feedback. I wasn't quite sure if I could just change the specs. This definitely answers that question. So we'd just put the # disable rubocop line length limit comment in the files where the error is?

Reply:
yes, or reformat the line to move it into multiple lines